pavelib/utils/test/utils.py

```diff
@@ -33,10 +33,17 @@ def clean_dir(directory):
 
 
 @task
-def clean_reports_dir():
+@cmdopts([
+    ('skip_clean', 'C', 'skip cleaning repository before running tests'),
+])
+def clean_reports_dir(options):
     """
     Clean coverage files, to ensure that we don't use stale data to generate reports.
     """
+    if getattr(options, 'skip_clean', False):
+        print('skipping')
```

Comment:
This message looks like it could use more explanation in context.
Here's how it shows up in the [console output.](https://test-jenkins.testeng.edx.org/job/edx-platform-test-suite/403/console)
```
---> pavelib.js_test.test_js
---> pavelib.prereqs.install_node_prereqs
Node prereqs unchanged, skipping...
---> pavelib.utils.test.utils.clean_reports_dir
skipping

Setting up for all javascript
find /home/jenkins/workspace/edx-platform-test-suite/reports/javascript -type f -delete
---> pavelib.assets.compile_coffeescript
```